### DISCUSSION TITLE
About skill development- course -github

### DISCUSSION THREAD

**Discussion by KalharaBatangala** [skills/.github#574] [Introduction to GitHub]
This is a very beginner friendly course whoever expect to start learning github and it guides very well step by step.

**Comment by hellogirlie** [comment]
> This is a very beginner friendly course whoever expect to start learning github and it guides very well step by step.

I'm sure you'll find out I'm a beginner in the process of learning the basic concepts of how to build a website and the basic knowledge of how to build a blog and then learn how to make money from it and how to build an online website affiliate marketing website that I can use to build my website and extra income from it and make money on my own site so that I can build my brand and get my product from the top merchants in merchandise and then sell and the website will inspire me and inform me and influence others about how much product development is possible and how much money that is possible for the website to build and how much of a business will be made from that website and then you can make money from the website
